cli/internal/fs/turbo_json.go

```diff
@@ -20,17 +20,26 @@ const (
 
 var defaultOutputs = []string{"dist/**/*", "build/**/*"}
 
-// TurboJSON is the root turborepo configuration
-type TurboJSON struct {
+type rawTurboJSON struct {
 	// Global root filesystem dependencies
 	GlobalDependencies []string `json:"globalDependencies,omitempty"`
+	// Global env
+	GlobalEnv []string `json:"globalEnv,omitempty"`
 	// Pipeline is a map of Turbo pipeline entries which define the task graph
 	// and cache behavior on a per task or per package-task basis.
 	Pipeline Pipeline
 	// Configuration options when interfacing with the remote cache
 	RemoteCacheOptions RemoteCacheOptions `json:"remoteCache,omitempty"`
 }
 
+// TurboJSON is the root turborepo configuration
+type TurboJSON struct {
+	GlobalFileDependencies []string
+	EnvVarDependencies     []string
```

Comment:
(request for change)

Lets make this match key name.

```suggestion
	GlobalDeps []string
	GlobalEnv  []string
```